autobuild/tfchain.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -ex
+
+apt-get update
+apt-get install git gcc wget -y
+
+# make output directory
+ARCHIVE=/tmp/archives
+FLIST=/tmp/flist
+mkdir -p $ARCHIVE
+
+# install go
+wget https://dl.google.com/go/go1.10.linux-amd64.tar.gz
+tar -C /usr/local -xzf go1.10.linux-amd64.tar.gz
```

Comment:
Perhaps the tar filename should be a variable here?

Reply:
this is just installing go, I don't get what you mean tbh

Reply:
He means (I guess) using a variable, so you can update the url without changing the tar command, like:
```bash
gofile=go1.10.linux-amd64.tar.gz
wget wget https://dl.google.com/go/${gofile}
tar -C /usr/local -xzf ${gofile}
```

Reply:
what @maxux said